modules/python/package/cv2/__init__.py

```diff
@@ -179,3 +179,9 @@ def load_first_config(fnames, required=True):
 
 
 bootstrap()
+
+# extra imports for a proper autocomplete work in IDE
+from . import gapi
+from . import mat_wrapper
+from . import misc
+from . import utils
```

Comment:
OpenCV has flexible module architecture (with custom set of modules), so such attempt of using hard-coded names just breaks the current design.
You can do that in `opencv-python` which uses fixed pre-defined build configuration, but it is not suitable for this upstream repository.

Reply:
I thought the idea was to keep one `__init__.py` file in `opencv` repository, and not to overwrite it in `opencv-python` repository. But okay, I'm closing this PR then.